.travis.yml

```diff
@@ -81,6 +81,18 @@ before_install:
 
   - pip install --upgrade pip setuptools
 
+  # Install  plugins for ui tests
+  - if [ "$TOXENV" = "ui" ]; then
```

Comment:
@ivan-c, is this where I should put this stuff? I was thinking either here or Dockerfile.build, but the Dockerfile seems to general for this specific case.

Reply:
Sorry, I should've been more clear and given you more background. 

You shouldn't need to make any changes to this file, Sauce Labs uses a ["remote" web driver connection](https://github.com/SeleniumHQ/selenium/wiki/RemoteWebDriver), through [a tunnel established by the Sauce Labs plugin for TravisCI](https://docs.travis-ci.com/user/sauce-connect/).

The "geckodriver" is Firefox-specific, and only needed when running tests locally (the remote web driver is included in selenium, but the gecko driver isn't :/)

FYI If you ever see TravisCI complaining about the missing geckodriver, it's likely that the Sauce Labs tunnel/connection failed and the `ui` job needs to be restarted

Reply:
TN-1468 <- @drryanjames please add "TN-1468 " to the front of your next commit comment

Reply:
@mcjustin I created a new PR with the Jira ticket in the title.

https://github.com/uwcirg/true_nth_usa_portal/pull/2602